### DISCUSSION TITLE
Deploy one application in the root context '/'

### DISCUSSION THREAD

**Discussion by Tirex** [Budibase/budibase#6605] [Ideas]
Use case:
I have a static website on domain.com with basic information about the product.
I created only one app in budibase and I want to deploy it in a root context on http://app.domain.com/ and continue to configure it with builder admin.

It's ok for me to exclude some context in the app for admin purposes, for example, 'http://app.domain.com/builder/',

**Comment by melohagan** [comment]
Hi @Tirex 

It should be possible through the use of the reverse proxy configuration: https://docs.budibase.com/docs/reverse-proxy

With that said, it is not recommended because it would interfere with the portal functionality, such as login/logout, etc.
